.vscode/cspell.json

```diff
@@ -858,6 +857,12 @@
         "jsonify",
         "ints"
       ]
+    },
+    {
+      "filename": "sdk/cognitiveservices/azure-cognitiveservices-vision-face/azure/cognitiveservices/vision/face/models/*.py",
+      "words": [
+        "responsed"
```

Comment:
Seeing as the file where this is misspelled is generated, I think this is the right course of action. Thanks for your contribution!